### ISSUE TITLE
Continued expression indentation

### ISSUE THREAD

**Issue by DeX3** [mooz/js2-mode#374]
Hey,

I write a lot of code using promises, so lots of code looks like this:

```
doSomething( function() {
   // ...
} )
.then( function() {
   // ...
} )
.then( function() {
  // ...
} );
```

However, with the default settings, `js2-mode` will indent this as:

```
doSomething( function() {
   // ...
} )
  .then( function() {
   // ...
  } )
  .then( function() {
  // ...
  } );
```

I've found lots of options related to indentation but I wasn't able to find one that controls this aspect. I'm fairly new to emacs (coming from vim) so I'm having a bit of a hard time digging through the source (although I'm getting there - slowly :))

In short, is there a way to make `js2-mode` not indent continuing expressions?

**Comment by dgutov** [comment]
If you're using Emacs 25 (and thus the indentation engine of `js-mode`), try:

```el
(setq js-expr-indent-offset (- js-indent-level))
```

**Comment by DeX3** [comment]
That does the trick thanks 👍 . Now I only have to make it work with editorconfig ;)
